### DISCUSSION TITLE
Show failed contact request attempt as a failed activity

### DISCUSSION THREAD

**Discussion by ilmotta** [status-im/status-legacy#16653] [General]
# Feature Issue

### User Story

>As a user
>When I send a contact request and it fails
>I want to see a new activity for the failed attempt

### Description

This feature is part of a larger effort to create a new Activity Center. Please, reach out to members of the UX team before starting this work in case there are important details not apparent in Figma.

- [Figma - Activity Center user journeys](https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=639%3A21926)

### Notes

- Should the user be able to retry? If so, how many times?
- It's not clear what exactly are the conditions for a contact request to fail. Consider evaluating these conditions before anything else.

**Comment by alwx** [comment]
Linked to https://github.com/status-im/status-mobile/pull/14853

Lacks a definition of failure (in addition, we retry to send a message automatically). Can you update on what we should consider as a failure?

**Comment by ilmotta** [comment]
@alwx: As I mentioned in the issue description a long time ago, it wasn't clear to me what were the failure conditions, what would trigger them, how the app should behave, etc. See the `Notes` section. I only very lightly captured what I saw on Figma, but the issue has always been very open-ended ;)

Given that the app's behavior is open for interpretation, I believe you'll need to talk to designers to understand what exactly we're trying to accomplish from the user's perspective. After that, it should be easier for you to define the technical specs + implementation plan.

You'll probably need to update this issue, because I think it's outdated or even incorrect.

**Comment by cammellos** [comment]
I believe this requires more discussion, moving to discussions for now
